includes/Admin/ReviewNotice.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace WeDevs\WeMail\Admin;
+
+/**
+ * Shiw admin notice.
+ */
+class ReviewNotice {
+
+    private static $instance;
+    public $time_based_review;
+    public $day_count;
+    public $campaign_count;
+
+    public static function instance() {         if ( ! isset( self::$instance ) ) {
+            self::$instance = new ReviewNotice();
+            self::$instance->boot();
+	}
+
+        return self::$instance;
+    }
+
+    /**
+     * Boot the notice
+     */
+    private function boot() {
+        add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+    }
+
+    public function enqueue_assets() {
+        wp_register_style( 'wemail-review-notice-style', WEMAIL_ASSETS . '/css/reviewNotice.css', false, filemtime( WEMAIL_PATH . '/assets/css/reviewNotice.css' ) );
+        wp_register_script( 'wemail-review-notice-script', WEMAIL_ASSETS . '/js/admin-review-notice.js', [ 'jquery' ], filemtime( WEMAIL_PATH . '/assets/js/admin-review-notice.js' ), true );
+
+        wp_enqueue_style( 'wemail-review-notice-style' );
+        wp_enqueue_script( 'wemail-review-notice-script' );
+    }
+
+    /**
+     * Notice markup to notify to connect site in weMail
+     *
+     * @return void
+     */
+    public function connect_review_notice_html() {         ?>
+        <div class="notice wemail-review-notice-flex-container is-dismissible">
+            <div class="wemail-review-notice-logo">
+                <img src="
+                <?php
+                if ( $this->time_based_review ) {
```

Comment:
Look, here just need to apply conditions on file names `time_based_review` and `campaign_based_review`. You can do it by the ternary operator.